deploy/ansible/create-cluster.yml

```diff
@@ -9,9 +9,25 @@
 - hosts: localhost
   connection: local
   gather_facts: True
+
   vars_files:
     - "../profiles/{{ profile }}.yml"
-    - "{{ lookup('env', 'CREDENTIAL_SECRETS') }}"
+
+  vars:
+    bucket: "{{ bucket }}"
```

Comment:
no need to put this vars to a separate section, should be in Profile with the rest of parameters.
bucket name variable should be more specific e.g secrets_bucket

Reply:
Main idea was the same as for https://github.com/legion-platform/legion/pull/176#discussion_r197151831 comment